addons/event_sale/models/event.py

```diff
@@ -77,8 +112,10 @@ def _default_product_id(self):
         default=_default_product_id)
     registration_ids = fields.One2many('event.registration', 'event_ticket_id', string='Registrations')
     price = fields.Float(string='Price', digits='Product Price')
+    beginning = fields.Date(string="Sales Start")
     deadline = fields.Date(string="Sales End")
     is_expired = fields.Boolean(string='Is Expired', compute='_compute_is_expired')
+    is_available = fields.Boolean(string='Is Available', compute='_compute_is_available')
```

Comment:
It is not really an absolute availability ... sale_date_open ? sale_date_availability ?